index.css

```diff
@@ -0,0 +1,169 @@
+@import url("https://fonts.googleapis.com/css2?family=Patrick+Hand&family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Mali:wght@200&family=Patrick+Hand&family=Roboto&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Mali", cursive;
+}
+
+:root {
+  --bgcolor: #edf2f4;
+  --gray: #8d99ae;
+  --primary-color: #2b2d42;
+  --white: #ffffff;
+}
+
+body {
+  background-color: var(--bgcolor);
+  width: 100%;
+}
+
+.quiz-home-page {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 4rem;
+}
+
+.home-page-title {
+  font-family: "Patrick Hand", cursive;
+  font-size: 3rem;
+  color: var(--gray);
+}
+
+.navbar {
+  padding: 0px;
+  background-color: var(--primary-color);
+  color: var(--white);
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.nav-link {
+  color: var(--white);
+  font-size: 1rem;
+  border-bottom: none;
+  transition: border-bottom 3s;
+}
+
+.nav-link:hover {
+  color: var(--white);
+  border-bottom: 2px solid var(--white);
+}
+
+.nav-logo {
+  color: var(--white);
+  font-size: 1.8rem;
+  font-family: "Mali", cursive;
+}
+
+.nav-logo:hover {
+  color: var(--gray);
+  border: none;
+}
+
+.explore-btn {
+  background-color: var(--primary-color);
+  color: var(--white);
+  border-top-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+  margin: 1rem;
+}
+
+.center-text {
+  text-align: center;
+}
+
+.heading-font {
+  font-family: "Patrick Hand", cursive;
+}
+
+.featured-categories {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 1rem;
+}
+
+.quiz-categories {
+  margin-top: 4rem;
+}
+
+.feature-category-item {
+  margin: 2rem 0 4rem 0;
+  border: 1px solid transparent;
+  border-radius: 10px;
+  padding: 4rem;
+  color: var(--white);
+  font-weight: 600;
+  font-size: 1.5rem;
+  width: 15rem;
+  height: 10rem;
+  text-align: center;
+  box-shadow: 0 3px 10px rgba(211, 209, 209, 0.2);
+  background: linear-gradient(90deg, #3d5a80 0%, #98c1d9 100%);
+  opacity: 0.9;
+  cursor: pointer;
+}
+
+.feature-category-item:hover {
+  opacity: 1;
+}
+
+.footer {
+  background-color: var(--primary-color);
+  color: var(--white);
+  padding: 1rem;
+}
+
+.footer-items {
+  display: flex;
```

Comment:
If flex property is repeated often, either you can declare a variable or separate class name in util file.

Reply:
done